apps/frontend/src/components/global/groups/Users.vue

```diff
@@ -173,15 +173,22 @@ export default class Users extends Vue {
     this.editedUserID = '0';
   }
 
-  deleteUserConfirm(): void {
+  @Emit()
+  deleteUserConfirm(): boolean {
+    let saveable = true;
+    const userToDelete = this.currentUsers.indexOf(this.getEditedUser());
+    if (
+      this.currentUsers[userToDelete].groupRole === 'owner' &&
+      this.numberOfOwners() <= 1
```

Comment:
we've been using `< 2` everywhere else though I agree that `<= 1` is clearer but let's keep it consistent with the `< 2` unless you wanna go and fix it everywhere